ietf/doc/views_search.py

```diff
@@ -307,6 +307,32 @@ def cached_redirect(cache_key, url):
 
     return cached_redirect(cache_key, urlreverse('ietf.doc.views_search.search') + search_args)
 
+def ad_dashboard_group_type(doc):
+    # Return group type for document for dashboard.
+    # If doc is not defined return list of all possible
+    # group types
+    if not doc:
+        return ('I-D', 'RFC', 'Conflict Review', 'Status Change', 'Charter')
+    if doc.type.slug=='draft':
+        if doc.get_state_slug('draft') == 'rfc':
+            return 'RFC'
+        elif doc.get_state_slug('draft') == 'active' and doc.get_state_slug('draft-iesg') and doc.get_state('draft-iesg').name in ('RFC Ed Queue Internet-Draft'):
```

Comment:
Should that be `...in ('RFC Ed Queue', 'Internet-Draft')`

Reply:
This looks like a real bug. Please address it, and consider the other feedback.